llm/ext_server/server.cpp

```diff
@@ -2762,6 +2762,23 @@ int wmain(int argc, wchar_t **wargv) {
 int main(int argc, char **argv) {
 #endif
 
+    // Short circuit for requirements reporting
```

Comment:
Why add these to server.cpp (I thought this PR would be more relevant to the Go stuff going forward)?

Reply:
removed